dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/TrackerEnrollmentsExportControllerTest.java

```diff
@@ -64,6 +78,18 @@ class TrackerEnrollmentsExportControllerTest extends DhisControllerConvenienceTe
 
     private ProgramInstance programInstance;
 
+    private TrackedEntityAttribute trackedEntityAttribute;
+
+    private Relationship relationship;
+
+    private ProgramStage programStage;
+
+    private ProgramStageInstance programStageInstance;
+
+    private DataElement dataElement;
+
+    private static final String attributeValue = "value";
```

Comment:
Not sure if we have a rule for that but generally constant names are all uppercase

Reply:
IntelliJ also does this by default when extracting a constant in its refactoring.